include/os/linux/kernel/linux/mod_compat.h

```diff
@@ -139,4 +139,9 @@ enum scope_prefix_types {
 	MODULE_PARM_DESC(name_prefix ## name, desc)
 /* END CSTYLED */
 
+#define	ZFS_MODULE_DESCRIPTION(s) MODULE_DESCRIPTION(s)
+#define	ZFS_MODULE_AUTHOR(s) MODULE_AUTHOR(s)
+#define	ZFS_MODULE_LICENSE(s) MODULE_LICENSE(s)
+#define	ZFS_MODULE_VERSION(s) MODULE_VERSION(s)
```

Comment:
If we add no-op versions of these to `include/sys/zfs_context.h` like `EXPORT_SYMBOL` eventually we'll be able to drop the kernel guard.